keyboards/handwired/bdn9_ble/rules.mk

```diff
@@ -0,0 +1,35 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+F_CPU = 8000000
```

Comment:
```suggestion
```

Reply:
Why should this line be removed?

Reply:
The suggestion above adds it before the bootloader section.